core/src/main/java/com/graphhopper/routing/QueryGraph.java

```diff
@@ -694,6 +702,9 @@ private boolean isInitialized() {
     }
 
     @Override
+    /**
+     * @see QueryGraph
```

Comment:
The new class comment also describes what the lookup methods are doing, so I made a reference here.